src/sentry/notifications/notifications/base.py

```diff
@@ -85,7 +86,7 @@ def template_path(self) -> str:
         pass
 
     def get_recipient_context(
-        self, recipient: Team | User, extra_context: Mapping[str, Any]
+        self, recipient: Team | APIUser, extra_context: Mapping[str, Any]
```

Comment:
@corps are these still valid types now that I put the User up-class in here? Does that matter for now?

Reply:
This should still be fine to send out.  I validated that the code paths worked for APIUser and User simult.  There could be one or two exceptions where I made a very minor change to support APIUser, so just check function body.

If function body is unchanged, it's safe to keep and it means I verified it by hand working the same for both types.